frontend/src/app/main/data/workspace/transforms.cljs

```diff
@@ -765,6 +768,23 @@
                  (remove (fn [shape]
                            (and (ctl/layout-absolute? shape)
                                 (= frame-id (:parent-id shape))))))
+
+            frame-component
+            (ctn/get-component-shape objects frame)
+
+            shape-ids-to-detach
+            (loop [current-shape (first moving-shapes)
```

Comment:
you're reducing over `moving-shapes` maybe a `reduce` will be better

Reply:
changed